packages/peregrine/lib/talons/WishlistPage/useCreateWishlist.js

```diff
@@ -16,18 +49,36 @@ export const useCreateWishlist = () => {
         setIsModalOpen(false);
     }, []);
 
-    const handleCreateList = useCallback(data => {
-        // TODO create list mutation is not available yet
-        // Will be handled in PWA-989
-        console.log('Creating wish list with data: ', data);
-        setIsModalOpen(false);
-    }, []);
+    const handleCreateList = useCallback(
+        async data => {
+            try {
+                await createWishlist({
+                    variables: {
+                        name: data.name,
+                        visibility: data.visibility
+                    }
```

Comment:
Paired with mutation input change:

```suggestion
                    variables: {
                        input: data
                    }
```

Reply:
Done